diff --git a/ps1Runtime/CMakeLists.txt b/ps1Runtime/CMakeLists.txt
index d439eee..7b2f4cc 100644
--- a/ps1Runtime/CMakeLists.txt
+++ b/ps1Runtime/CMakeLists.txt
@@ -1,4 +1,6 @@
 find_package(SDL2 REQUIRED)
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
 find_package(OpenGL REQUIRED)
 
 # Runtime library (shared between exe and tests)
@@ -98,4 +100,5 @@ target_link_libraries(ps1Runtime
         SDL2::SDL2
         OpenGL::GL
         ps1Analyzer_lib
-)
+
+    Threads::Threads)
